From ca08b536c82c4b51b0336f9002ccc5b019189aab Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <ccecchi@apple.com>
Date: Wed, 20 May 2020 14:45:42 -0700
Subject: [PATCH] <rdar://problem/61766230> GoldenTokyoSideTwo: python-126#467
 has failed to build in install; make: *** [build] Error 1

Exclude /System/Volumes/Data/SWE from the paths that are considered
valid for a macOS SDK.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3b32ecc..9632d7d 100644
--- a/setup.py
+++ b/setup.py
@@ -63,7 +63,7 @@ def is_macosx_sdk_path(path):
     Returns True if 'path' can be located in an OSX SDK
     """
     return ( path.startswith('/usr/')
-                or path.startswith('/System/')
+                or path.startswith('/System/') and not path.startswith('/System/Volumes/Data/SWE/')
                 or path.startswith('/Library/') and not path.startswith('/Library/Caches/') )
 
 def find_file(filename, std_dirs, paths):
-- 
2.24.3 (Apple Git-128)

